go/vt/sqlparser/ast_permissions.go

```diff
@@ -595,9 +595,11 @@ type CreateUser struct {
 	PasswordOptions *PasswordOptions
 	Locked          bool
 	Attribute       string
+	Auth            AuthInformation
```

Comment:
Maybe just embed this type directly since the implementation is the same everywhere?